schemas/2017-04-01/Microsoft.Relay.json

```diff
@@ -125,7 +125,7 @@
         "type": {
           "type": "string",
           "enum": [
-            "Microsoft.Relay/namespaces/hybridConnection"
+            "Microsoft.Relay/namespaces/hybridConnections"
```

Comment:
>hybridConnections [](start = 40, length = 17)

Please update the tests for these changes.

Reply:
Updated tests